### PR TITLE
Update qos_ecn_config_test.go

### PR DESCRIPTION
**Pull request by saarvind6481**
cisco-1.3-uprev

### CODE REVIEW

File: feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go [hunk 1]
```diff
@@ -256,11 +256,13 @@ func testCiscoECNConfig(t *testing.T) {
...
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
-	queueName := []string{"a_NC1", "b_AF4", "c_AF3", "d_AF2", "e_AF1", "f_BE0", "g_BE1"}
+	queueName := []string{"NC1", "AF4", "AF3", "AF2", "AF1", "BE0", "BE1"}
 
-	for _, queue := range queueName {
+	for i, queue := range queueName {
 		q1 := q.GetOrCreateQueue(queue)
 		q1.Name = ygot.String(queue)
+		queueid := 7 - i
```
Comment (guoshiuan) [inline feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go:264]: 7 is magic number to me.. 
If it is a fixed value, please make it a constant with a meaning name.
If it is the size of the "queueName" list, please use len().

File: feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go [hunk 2]
```diff
@@ -278,64 +280,64 @@ func testCiscoECNConfig(t *testing.T) {
 		desc:         "scheduler-policy-BE1",
 		sequence:     uint32(1),
 		priority:     oc.Scheduler_Priority_UNSET,
-		inputID:      "g_BE1",
+		inputID:      "BE1",
 		inputType:    oc.Input_InputType_QUEUE,
 		weight:       uint64(1),
-		queueName:    "g_BE1",
+		queueName:    "BE1",
```
Comment (guoshiuan) [inline feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go:286]: Are inputID and queueName always the same? If so, we can combine them?

File: feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go [hunk 3]
```diff
@@ -365,9 +367,9 @@ func testCiscoECNConfig(t *testing.T) {
 	}{
 		ecnEnabled:                true,
 		dropEnabled:               false,
-		minThreshold:              uint64(80000),
-		maxThreshold:              uint64(1000000),
-		maxDropProbabilityPercent: uint8(1),
+		minThreshold:              uint64(8005632),
+		maxThreshold:              uint64(8011776),
+		maxDropProbabilityPercent: uint8(100),
```
Comment (guoshiuan) [inline feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go:372]: Does this mean we allow 100% traffic loss?

File: feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go (hunk 1 above)
Comment (saarvind6481) [reply feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go]: fixed

File: feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go (hunk 2 above)
Comment (saarvind6481) [reply feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go:285]: fixed

File: feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go [hunk 4]
```diff
@@ -256,11 +256,13 @@ func testCiscoECNConfig(t *testing.T) {
...
 	q := d.GetOrCreateQos()
-	queueName := []string{"a_NC1", "b_AF4", "c_AF3", "d_AF2", "e_AF1", "f_BE0", "g_BE1"}
+	queueName := []string{"NC1", "AF4", "AF3", "AF2", "AF1", "BE0", "BE1"}
 
-	for _, queue := range queueName {
+	for i, queue := range queueName {
 		q1 := q.GetOrCreateQueue(queue)
 		q1.Name = ygot.String(queue)
+		queueid := len(queueName) - i
+		q1.QueueId = ygot.Uint8(uint8(queueid))
```
Comment (guoshiuan) [inline feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go:266]: Please remove the blank line @ 266.

**Comment by guoshiuan** [comment]
b/285614685 has the internal test results.

**Review by guoshiuan — APPROVED** [review]

**Pull request merged**